<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>My_Test</class>
 <widget class="QWidget" name="My_Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 255);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>780</width>
     <height>361</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>编号</string>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>85</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>箱号</string>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电压1(V)</string>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电压2(V)</string>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电流/电压</string>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>状态</string>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>通讯</string>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="4" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="5" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="6" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="7" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="8" column="0">
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item row="8" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
   <item row="9" column="0">
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item row="9" column="1">
    <property name="text">
     <string>1ALE</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>370</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <widget class="MyButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyButton" name="pushButton_2">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyButton" name="pushButton_3">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>自动翻页</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>387</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>当前报警点数:0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>当前故障点数:0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header location="global">mybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
